### ISSUE TITLE
Question About Legacy.com API Usage in the Project

### ISSUE THREAD

**Issue by orogenicbelt** [littlehoushontheprairie/check-obituaries-alert#5]
Hi Laura

I stumbled on this and am intrigued. I am interested in this tool and am curious how you discovered the API https://www.legacy.com/api/_frontend/? Is it publicly documented or did you find it via reverse engineering or similar means? I'm curious about the APIs usage guidelines or potential limitations. 

Thanks,
Mike

**Comment by littlehoushontheprairie** [comment]
Hi. Sorry for the late response. I just reverse engineered their endpoints by monitoring the developer console and comparing to the rendered page. It usually has some kind of tie in. For this one, there's a few things I had to watch out for:
1. Look for State/Providence Codes which I validated on the page. 
2. Make sure the api wasn't using some weird logic. 
3. Trial and error. I remember I had to pass in the user agent since my request couldn't go through and so I just use a typical iOS one.
